### PR TITLE
Update alert-policies.md

### PR DESCRIPTION
**Pull request by tshepang**
reduce some repetition... the paragraph mentions things listed immediately below

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/compliance/alert-policies.md [hunk 1]
```diff
@@ -270,19 +270,19 @@ After alerts have been generated and displayed on the **View alerts** page in th
...
 
   - A link to an activity list that includes an item for each activity that was performed that triggered the alert. Each entry in this list identifies when the activity occurred, the name of actual operation (such as "FileDeleted"), and the user who performed the activity, the object (such as a file, an eDiscovery case, or a mailbox) that the activity was performed on, and the IP address of the user's computer. For malware-related alerts, this links to a message list.
 
-  - The name (and link to) of the corresponding alert policy.
+  - The name (and link) of the corresponding alert policy.
 
 - **Suppress email notifications.** You can turn off (or suppress) email notifications from the flyout page for an alert. When you suppress email notifications, Microsoft won't send notifications when activities or events that match the conditions of the alert policy. But alerts will be triggered when activities performed by users match the conditions of the alert policy. You can also turn off email notifications by editing the alert policy.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/alert-policies.md:287]:
```suggestion
- **Suppress email notifications.** You can turn off (or suppress) email notifications from the flyout page for an alert. When you suppress email notifications, Microsoft won't send notifications when activities or events that match the conditions of the alert policy occur. But alerts will be triggered when activities performed by users match the conditions of the alert policy. You can also turn off email notifications by editing the alert policy.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @tshepang, kindly also commit the additional suggested change. Many thanks.

**Comment by markjjo** [comment]
@tshepang PR merged. Thanks for making these revisions.

**Pull request merged**